wagtailio/project_styleguide/templates/patterns/components/streamfields/cards/related_page_card_block.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+<a href="{{ value.url }}" class="logo-card {% if classes %}{{ classes }}{% endif %}">
```

Comment:
Suggestion: add a comment in `logo_card_block.html` that it's also used here.

Reply:
Thanks, done in 10518e7